sensorserviceutils/sensorserviceutils.go

```diff
@@ -278,51 +399,64 @@ func (p parser) extractActiveConnInfo() error {
+					SensorNumber:   -1,
+					PendingFlush:   -1,
+					SamplingPeriod: -1,
+					BatchingPeriod: -1,
+					HasRecord:      false,
 				}
 			}
 		} else {
 			_, ok := connections[curConnNum]
 			if !ok {
-				p.parsingErrs = append(p.parsingErrs,
-					fmt.Errorf("no information regarding connection %v", curConnNum))
+				p.parsingErrs = append(p.parsingErrs, fmt.Errorf(
+					"%s: connection(%d): no information for this connection",
+					parseConnErrStr, curConnNum))
 				continue
 			}
 			if lineTwo, result := historianutils.SubexpNames(connLineTwoRE, line); lineTwo {
 				connections[curConnNum].OperatingMode = result["connMode"]
 			} else if lineThree, result := historianutils.SubexpNames(connLineThreeRE, line); lineThree {
 				uid, err := strconv.Atoi(result["uid"])
 				if err != nil {
-					p.parsingErrs = append(p.parsingErrs,
-						fmt.Errorf("active connection %d: cannot parse uid %v:%v",
-							curConnNum, result["uid"], err))
+					p.parsingErrs = append(p.parsingErrs, fmt.Errorf(
+						"%s: connection(%d): error parsing uid %v:%v",
+						parseConnErrStr, curConnNum, result["uid"], err))
 					continue
 				}
 				connections[curConnNum].UID = int32(uid)
 				connections[curConnNum].PackageName = result["packageName"]
 			} else if lineFour, result := historianutils.SubexpNames(connLineFourRE, line); lineFour {
-				sensorNumber, err := strconv.ParseInt(result["sensorNumber"], 0, 32)
+				n, err := strconv.ParseInt(result["sensorNumber"], 0, 32)
 				if err != nil {
-					p.parsingErrs = append(p.parsingErrs,
-						fmt.Errorf("active connection %d: cannot parse sensor handle %v:%v",
-							curConnNum, result["sensorNumber"], err))
+					p.parsingErrs = append(p.parsingErrs, fmt.Errorf(
+						"%s: connection(%d): error parsing sensorNumber %v:%v",
+						parseConnErrStr, curConnNum, result["sensorNumber"], err))
 					continue
 				}
-				sensor := int32(sensorNumber)
-				connections[curConnNum].SensorNumber = sensor
+				sensorNumber := int32(n)
+				connections[curConnNum].SensorNumber = sensorNumber
 				pendingFlush, err := strconv.Atoi(result["pendingFlush"])
 				if err != nil {
-					p.parsingErrs = append(p.parsingErrs,
-						fmt.Errorf("active connection %d: cannot parse pendingFlush %v:%v",
-							curConnNum, result["uid"], err))
+					p.parsingErrs = append(p.parsingErrs, fmt.Errorf(
+						"%s: connection(%d): error parsing pendingFlush %v:%v",
+						parseConnErrStr, curConnNum, result["pendingFlush"], err))
 					continue
 				}
 				connections[curConnNum].PendingFlush = int32(pendingFlush)
+				if activeSensor, exist := p.activeSensors[sensorNumber]; exist {
+					connections[curConnNum].SamplingPeriod = activeSensor.samplingPeriod
+					connections[curConnNum].BatchingPeriod = activeSensor.batchingPeriod
+				} else {
+					p.sensorErrs = append(p.sensorErrs, fmt.Errorf(
+						"[Active Connection]: connection(%d): the sensor(%d) is not active according to the sensor device section",
```

Comment:
line too long

Reply:
I have made two new commits to fix the issues regarding comments, length for each line and units for variable names. I will make sure to double-check these issues when I make more new PRs later. 
I did not strictly follow the 80-character length limit for the sensorserviceutils_test.go file since each line of the input strings is usually long and it makes more sense to present these lines one by one.

Reply:
Regarding the 80-character length limit, I set the tab size to be 4 on my local editor, but github has a default configuration of 8. I will see if I can change that setting for the whole repo somewhere.